src/Symfony/Component/Serializer/Normalizer/DateIntervalNormalizer.php

```diff
@@ -34,8 +34,6 @@ public function __construct(array $defaultContext = [])
     }
 
     /**
-     * {@inheritdoc}
-     *
```

Comment:
These comment should be removed in that PR

Reply:
fixed